x/fungible/types/message_deploy_fungible_coin_zrc20.go

```diff
@@ -50,7 +50,11 @@ func (msg *MsgDeployFungibleCoinZRC20) ValidateBasic() error {
 		return sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress, "invalid creator address (%s)", err)
 	}
 	if msg.GasLimit < 0 {
-		return sdkerrors.Wrapf(sdkerrors.ErrInvalidGasLimit, "invalid gas limit (%s)", err)
+		return sdkerrors.Wrapf(sdkerrors.ErrInvalidGasLimit, "invalid gas limit")
+	}
+
+	if msg.Decimals > 255 {
```

Comment:
Would `if msg.Decimals > 77` be a more realistic number in here as ERC20 standard uses `uint256` `(2 ** 256)-1 ≈ 10 ** 77` ?

Reply:
@lukema95 what do you think

Reply:
I agree with you, `if msg.Decimals > 77` would be more accurate, I will change it later.